cves/kernel/CVE-2016-2184.yml

```diff
@@ -84,14 +88,16 @@ fixes_instructions: |
 
   Place any notes you would like to make in the notes field.
 fixes:
-- commit:
+- commit: '0f886ca12765d20124bd06291c82951fd49a33be'
   note:
+    create_fixed_stream_quirk() may cause a NULL-pointer dereference by 
+    accessing the non-existing endpoint when a USB device with a malformed USB
+    descriptor is used. This patch avoids it simply by adding a sanity check of
+    bNumEndpoints before the accesses.
```

Comment:
This probably shouldn't reference specific functions and variable names, but instead describe them at a higher-level for the reader.